views/partials/alert.ejs

```diff
@@ -0,0 +1,9 @@
+<% for (i=0; i < data.length; i++) { %>
+    <section class="card">
+
+        <h4><a href="<%= data[i].uri %>"><%= data[i].title %> until <%= moment.utc(data[i].expires * 1000).format("h:mm a [on] dddd, MMMM Do") %></a></h4>
+        <p class="status-subtitle"><%= data[i].description.substr(0,139) %>...[alert continues]</p>
```

Comment:
Subjective advice, up to you:

change "[alert continues]" to 
… <a>continued</a>